src/lib/caching/structures/channels/GuildChannel.ts

```diff
@@ -40,6 +41,12 @@ export abstract class GuildChannel extends Channel {
 	 */
 	public permissionOverwrites!: OverwriteStore;
 
+	/**
+	 * The {@link GuildChannelInviteStore invites} store for this channel.
+	 * @since 0.0.3
+	 */
+	public readonly invites!: GuildChannelInviteStore;
```

Comment:
You assert that invites exists on GuildChannel, but you haven't actually created the value where it does...